backend/api/processors/course_build_processor.py

```diff
@@ -91,18 +93,33 @@ def process_course_build(course_build_id):
             course_build.course_enrollment_url = new_course["course_enrolment_url"]
             course_build.course_enrollment_key = new_course["course_enrolment_key"]
             session.commit()
+
+            s3_bucket = settings.COURSES_CSV_S3_BUCKET
+            s3_key = settings.COURSES_CSV_S3_KEY
+            response = s3_client.get_object(Bucket=s3_bucket, Key=s3_key)
+            csv_data = response["Body"].read().decode("utf-8")
+
+            new_row = [course_build.course_id, course_build.school_district.name, 0]
+            csv_rows = list(csv.reader(csv_data.splitlines(), delimiter=','))
+            csv_rows.append(new_row)
```

Comment:
Check me if I'm wrong, but I think this code:

1. Makes a pretty strong assumption around the ordering + number of columns. If so, it might be nice to remove this tight coupling.
2. May break if, for whatever reason, a data entry happens to have a `\n` due to use of `splitlines()`

Something like the following may be a bit more robust on both points:

```python
            csv_rows = list(csv.DictReader(io.StringIO(csv_data)))
            csv_rows.append({
                "course_id": course_build.course_id,
                "district": course_build.school_district.name,
                "research_participation": 0
            })

            updated_csv_data = io.StringIO()
            csv_writer = csv.DictWriter(
                updated_csv_data,
                fieldnames=csv_rows[0].keys()
            )
            csv_writer.writeheader()
            csv_writer.writerows(csv_rows)
            s3_client.put_object(Bucket=s3_bucket, Key=s3_key,
                                 Body=updated_csv_data.getvalue().encode("utf-8"))
```